### ISSUE TITLE
Duplicate titles are not prevented when mass import functionality is used

### ISSUE THREAD

**Issue by BartChris** [kitodo/kitodo-production#4988]
While importing multiple prcoesses at once using the mass import feature in combination with a "processtitle"-attribute for the division to automatically generate the process title, no check for duplicate titles is taking place. 

This should probably be adressed in a similar way as this issue
https://github.com/kitodo/kitodo-production/issues/4900

**Comment by matthias-ronge** [comment]
What should be the behavior when a duplicate process title is found? Suppose, some of the processes have already been created, and then the application encounters a process whose title already exists ...

It would certainly be a bad solution if the mass import then aborted, one part was processed and one part was not, and you don't know how far it came and why it broke.

I can think of two sensible solutions:
- Either the process in question is simply skipped (and at best, this is also logged or reported somewhere)
- or the process is created anyhow, and given a suffix.

**Comment by BartChris** [comment]
The best way would probably be if this is reported to the user, that a generated processtitle was already created (which usually means that for the given Catalog identifier (e.g. PPN) a process has already been created). And then the user can chose if he or she wants to a) skip this process, b) append a suffix. I do not now if this could be implemented

**Comment by matthias-ronge** [comment]
If the mass import runs in the background, this must be specified before starting. Displaying a question from the background process is not reasonably possible within web applications.

**Comment by solth** [comment]
@BartChris this might have been solved by #5039
